src/styles/partials/project.sass

```diff
@@ -0,0 +1,88 @@
+.project
+    border: 0.5px solid #DADADA 
+    box-sizing: border-box
+    padding-left: 77px
+    position: relative
+    color: $text-black
+    margin-top: 24px
+
+    +screen-down(lg)
+        padding: 24px 14px 14px 14px
+
+
+    &:hover, &:focus, &:active
+        box-shadow: (0px 0px 64px rgba(0, 0, 0, 0.09))
+        cursor: pointer
+
+    &-featured
+
+        +screen-down(lg)
+            padding-top: 56px
+
+        &::before
+            content: 'Featured'
+            position: absolute
+            background-color: $text-accent
+            padding: 5px 75px
+            box-sizing: border-box
+            left: 2px
+            top: 2px
+            color: $text-grey
+
+            +screen-down(lg)
+                padding: 6px 0 
+                width: 100%
+                text-align: center
+
+    & -title
+        max-width: 310px
+
+        +screen-down(lg)
+            margin-top: 200px
+
+    .subtitle
+        font-size: 32px
+        margin-bottom: 24px
+
+        +screen-down(lg)
+            margin-bottom: 38px
+            font-size: 24px
+
+    .desc
+        
+        +screen-down(lg)
+            font-size: 16px
+            text-align: center
+
+
+    & img
+        padding: 2px
+        height: auto
+        max-width: 400px
+
+.img-wrapper
+    position: relative
+
+    +screen-down(lg)
+        margin-top: 64px
+
+    &::after
+        content: 'View Project'
+        position: absolute
+        background-color: #ffffff
+        padding: 10px 10px 10px 0px
+        box-sizing: border-box
+        left: -23px
+        top: 70px
+
+        +screen-down(lg)
+            left: 50%
+            transform: translateX(-50%)
+            top: -29px
+            padding: 10px 15px
+            
+.space-top
```

Comment:
After you'll move all the content to the appropriate section with projects, remove classes `.space-top` and `.space-bottom`. Set this space by paddings within the section.